x-pack/plugins/cases/server/connectors/cases/index.ts

```diff
@@ -56,7 +56,7 @@ export const getCasesConnectorType = ({
     config: CasesConnectorConfigSchema,
     secrets: CasesConnectorSecretsSchema,
   },
-  supportedFeatureIds: [UptimeConnectorFeatureId, AlertingConnectorFeatureId],
+  supportedFeatureIds: [UptimeConnectorFeatureId, AlertingConnectorFeatureId, 'siem'],
```

Comment:
Look like you can import `SecurityConnectorFeatureId` from the same file where `UptimeConnectorFeatureId` and `AlertingConnectorFeatureId` are imported from.